shopinvader_locomotive_reset_password/models/shopinvader_backend.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+from odoo.addons.queue_job.job import job
+
+
+class ShopinvaderBackend(models.Model):
+
+    _inherit = "shopinvader.backend"
+
+    password_validity = fields.Integer(
+        default=0,
+        string="Password Validity (days)",
+        help="Define here the user password validity (in days) on Website. "
+        "If the validity is passed, an email would be sent to reset it."
+        "To deactivate it, set it to 0.",
```

Comment:
I would set -1 to inactivate. some process on shopinvader already use this convention.

Reply:
euh... where ? And an integer cannot be negative.

Reply:
see there for e.g. https://github.com/shopinvader/odoo-shopinvader/blob/10.0/shopinvader/models/shopinvader_notification.py#L69

Reply:
Indeed

Reply:
But does it have sense of 0 days validity ?

Reply:
no indeed...